libs/dh/metering-point/feature-tabs/src/lib/processes-tab-content/processes-table/dh-processes-table.component.spec.ts

```diff
@@ -135,4 +136,30 @@ describe(DhProcessesTableComponent.name, () => {
       'check_circle'
     );
   });
+
+  it('Should be able to expand and collapse details by clicking on the process row', async () => {
+    await setup(testData);
+
+    const processRow = screen.getByTestId(
+      'processRow-' + succeededProcessId,
+      disableQuerySuggestions
+    );
+
+    const detailsRow = screen.getByTestId(
+      'detailsRow-' + succeededProcessId,
+      disableQuerySuggestions
+    );
+
+    // Verify that the success row is set to collapsed and has height 0
+    expect(detailsRow.classList.toString()).toContain('collapsed');
```

Comment:
nitpick: instead of using `.toString()` in this and following assertions, Jest has a helper matcher `toHaveClass`. E.g.

`expect(processRow).toHaveClass('expanded');`

Reply:
Neat!